Python/Product/Debugger/DkmDebugger/PyObjectNativeVisualizer.cs

```diff
@@ -73,6 +73,7 @@ internal class PyObjectNativeVisualizer : DkmDataItem {
                 return null;
             }
 
+            // TODO: Localization ?
             var pyEvalResult = new PythonEvaluationResult(objRef, "[Python view]")
```

Comment:
"[Python view]" appears in `\Python\Product\Debugger\DkmDebugger\PythonDkm.natvis`, so I think it's safer to not change it for now.